mmv1/templates/terraform/constants/disk.tmpl

```diff
@@ -43,6 +43,18 @@ func IsDiskShrinkage(_ context.Context, old, new, _ interface{}) bool {
 	return new.(int) < old.(int)
 }
 
+func matchImageLink(old string) ([]string) {
+	// 'old' is read from the API.
+	// It has the format 'https://www.googleapis.com/compute/v1/projects/(%s)/global/images/(%s)'
+	// Except if the image is created in different universes
+	matches := resolveImageLink.FindStringSubmatch(old)
+	if matches == nil {
```

Comment:
```suggestion
	// In alternate universes, it has the format https://compute.%s/compute/[a-z0-9]+/projects/(%s)/global/images/(%s)
	if matches == nil {
```

Reply:
Applied the comment to this line. But I prefer to keep the first if check since we only do resolveImageUniverseLink if resolveImageLink didn't match